cluster-autoscaler/core/scaledown/eligibility/eligibility_test.go

```diff
@@ -111,14 +114,32 @@ func TestFilterOutUnremovable(t *testing.T) {
 			scaleDownUnready: false,
 		},
 	}
+
+	allTestCases := testCases
+
+	// run all test cases again with `IgnoreDaemonSetsUtilization` set to true
```

Comment:
This will cause different test cases to have the same `desc`, making them harder to debug. For the sake of readability, I'd just have one long list of test cases to check, but if you have to generate them programatically like this, please update `desc` as well.

Reply:
Thank you for the suggestion. 

I have [started adding a suffix now](https://github.com/kubernetes/autoscaler/pull/5672/files#diff-e7f41c366e8f9e299ef7f726cb60c9fe2e6943fcdb6f2945a0df18d27f6bd015R152) like `<test-case-desc> IgnoreDaemonSetsUtilization=true` and `<test-case-desc> IgnoreDaemonSetsUtilization=false` and